### PR TITLE
modules: use go mod

### PR DESCRIPTION
**Pull request by moshe010**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module github.com/Mellanox/ipoib-cni
+
+go 1.13
+
+require (
+	github.com/containernetworking/cni v0.7.1
+	github.com/containernetworking/plugins v0.8.3
+	github.com/coreos/go-iptables v0.4.2
```
Comment (adrianchiris) [inline go.mod:8]: here there should be entries of projects that are directly used by ipoib-cni
i dont think this project uses all of these directly doesnt it ?

Comment (moshe010) [reply go.mod]: sorry forgot to run go mod tidy

**Comment by adrianchiris** [comment]
@moshe010 according to [1] there should be 1-2 files per version of a package used.

[1] https://golang.org/cmd/go/#hdr-Module_authentication_using_go_sum

**Comment by adrianchiris** [comment]
anyway, /lgtm

**Pull request merged**